### PR TITLE
test: Moves basic import tests to basic acc tests

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Move basic import tests to basic acc test.

This is already done in the other resources.

- Improves use of resources as both tests are the same so only create resources once.
- Improves code legibility reducing redundant code to set up the same test configuration.

Link to any related issue(s): CLOUDP-240949

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

**Review by EspenAlbert — APPROVED** [review]
Wow. Nice 👏

File: internal/service/alertconfiguration/resource_alert_configuration_test.go [hunk 1]
```diff
@@ -338,68 +326,10 @@ func TestAccConfigRSAlertConfiguration_importIncorrectId(t *testing.T) {
 	})
 }
 
-func TestAccConfigRSAlertConfiguration_importConfigNotifications(t *testing.T) {
```
Comment (marcosuma) [inline internal/service/alertconfiguration/resource_alert_configuration_test.go:341]: Are we losing this specific test? it seems there isn't a counterpart during acc_test (unless I am missing it)

File: internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go [hunk 2]
```diff
@@ -36,27 +36,6 @@ func TestAccBackupRSBackupSnapshotExportBucket_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "cloud_provider", "AWS"),
 				),
 			},
-		},
-	})
-}
-
-func TestAccBackupRSBackupSnapshotExportBucket_importBasic(t *testing.T) {
-	acc.SkipTestForCI(t)
```
Comment (marcosuma) [inline internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go:44]: what about these tests? what is the reason why are not being merged in the TestAcc?

File: internal/service/alertconfiguration/resource_alert_configuration_test.go (hunk 1 above)
Comment (lantoli) [reply internal/service/alertconfiguration/resource_alert_configuration_test.go:341]: it's joined to TestAccConfigRSAlertConfiguration_withNotifications

File: internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go (hunk 2 above)
Comment (lantoli) [reply internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go:44]: TestAccBackupRSBackupSnapshotExportBucket_importBasic is being joined to TestAccBackupRSBackupSnapshotExportBucket_basic

Comment (marcosuma) [reply internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go:44]: ah - I see now the diff has merged the two tests

**Review by marcosuma — APPROVED** [review]

**Pull request merged**